toasty/fits_tiler.py

```diff
@@ -167,6 +167,9 @@ def tile(
         else:
             self._tile_tan(cli_progress, parallel, **kwargs)
 
+        for img in self.coll.images():
+            self.builder.make_thumbnail_from_other(img)
+            break
```

Comment:
Stylistically, I like to have blank lines both before and after indenting constructs such as `for`, `if,` etc.